packages/schemas/src/seeds/sign-in-experience.ts

```diff
@@ -33,9 +33,11 @@ export const defaultSignInExperience: Readonly<CreateSignInExperience> = {
   signInMode: SignInMode.SignInAndRegister,
 };
 
-export const adminConsoleSignInMethods = {
-  username: SignInMethodState.Primary,
-  email: SignInMethodState.Disabled,
-  sms: SignInMethodState.Disabled,
-  social: SignInMethodState.Disabled,
+export const adminConsoleSignInExperience: CreateSignInExperience = {
+  ...defaultSignInExperience,
+  branding: {
+    style: BrandingStyle.Logo_Slogan,
```

Comment:
Why is this using `camel snake case`?

Reply:
maybe some legacy issue, defined in schema like this.